src/components/App.js

```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react';
 import { Dimensions, Platform } from 'react-native';
 import {
   Actions,
+  AppState,
```

Comment:
Why do you add this?
I can't find this component at react-native-router-flux.
If you want to use this, you have to move to react-native.